### ISSUE TITLE
With older git returns an error

### ISSUE THREAD

**Issue by thomas9911** [qgadrian/elixir_git_hooks#100]
with git version before 2.22

run:

```
> mix git_hooks.run all
error: unknown option `show-current'
```

I looked up on the internet and you can get the version with:

```sh
git rev-parse --abbrev-ref HEAD
```

as well

**Comment by thomas9911** [comment]
made a pull request https://github.com/qgadrian/elixir_git_hooks/pull/101

**Comment by qgadrian** [comment]
Fixed in #103
